clarify auto ip address (#3691)
getsentry/sentry-docs: src/platforms/common/enriching-events/identify-user.mdx

`ip_address`

> : The user's IP address. If the user is unauthenticated, Sentry uses the IP address as a unique identifier for the user. Sentry will attempt to pull this from the HTTP request data, if available. That might required `sendDefaultPii` set to `true` in the SDK options.
> You can set to `"{{auto}}"` to let Sentry infer the IP address from the connection between your app and Sentry's server. This is useful for client application such as a website, desktop and mobile.

Reviewer: ```suggestion
: The user's IP address. If the user is unauthenticated, Sentry uses the IP address as a unique identifier for the user. Sentry will attempt to pull this from the HTTP request data, if available. That might required `sendDefaultPii` set to `true` in the SDK options. If set to `"{{auto}}"`, Sentry will infer the IP address from the connection between your app and Sentry's server, which is useful for client applications such as a website, desktop, or mobile.
```
